### ISSUE TITLE
Taskbar Autohide Issue - Windows 11

### ISSUE THREAD

**Issue by khushrxnx** [ChanpleCai/SmartTaskbar#98]
**Describe the bug**

For a non-maximized window, when I click Start Menu to show up and then click it again to hide it, the Start Menu autohides and then pops up again. This is slightly annoying.

**To Reproduce**

1. Click Start Manu to bring to up
2. Click Start Manu to hide it
3. Observe the unnecessary autohide of the taskbar and feel the annoyance XD

**Expected behavior**

When you click the Start Menu the second time to hide it, it should not automatically hide and pop up, rather just stay there as a constant.  

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 11

**Comment by ChanpleCai** [comment]
Close the start menu or search, the taskbar will be forcibly hidden once. This is a rule designed by Microsoft since the later version of Windows 10. It is also the secret of why SmartTaskbar can control the display and hide of the taskbar (in auto-hide mode).
If I want to solve this problem, I need to hook the taskbar to block its default behavior. I haven’t done this yet for two reasons. The first is that I’m not sure what side effects will be after doing this (the Windows team forces the taskbar to hide after closing the start menu, there should be a reason), and the second is that The software will become more complicated. In order to fix an issue, add several new components, just to ensure that the taskbar will not be hidden when the start menu is closed.
But I actually tried to fix this problem two years ago. At that time, no new version was released because of the above two considerations. After experimenting to prevent the default hiding, the behavior of the taskbar did appear to be abnormal. In fact, I am still struggling so far. I always hope that SmartTaskbar is a small and beautiful application, but there will always be one or two unsolvable bugs that force it to become complex and ugly.
I may fix this issue later, but it is a bit beyond my personal ability at present.
